parser-typechecker/src/Unison/Hashable.hs

```diff
@@ -14,6 +14,7 @@ data Token h
   | Bytes !ByteString
   | Int !Int64
   | Text !Text
+  | Char !Char
```

Comment:
Let's not introduce a new token type. Hashable instance for `Char` can produce `Int` token from the codepoint of the `Char`.

Reply:
Done.